README.md

```diff
@@ -1,3 +1,44 @@
-# IDR metadata
+# IDR studies
 
 [![Published screens](https://img.shields.io/badge/dynamic/json.svg?label=Published%20&url=http%3A%2F%2Fidr.openmicroscopy.org%2Fapi%2Fv0%2Fm%2Fscreens%2F%3Flimit%3D0&query=meta.totalCount&colorB=blue&suffix=%20high%20content%20screens)](https://idr.openmicroscopy.org) [![Published experiments](https://img.shields.io/badge/dynamic/json.svg?label=Published%20&url=http%3A%2F%2Fidr.openmicroscopy.org%2Fapi%2Fv0%2Fm%2Fprojects%2F%3Flimit%3D0&query=meta.totalCount&colorB=blue&suffix=%20experiments)](https://idr.openmicroscopy.org)
+
+All metadata associated with published studies in IDR is managed.
+
+## Study name 
+
+After acceptance, IDR studies must be named as `idr<NNNN>-<name>-<description>`
+where `idr<NNNN>` is the accession number of the study using an incremental
+four digits integer, `<name>` is the name of one of the authors associated
```

Comment:
IIRC we discussed allowing a consortium name instead of an author name, did we ever make a decision on that?

Reply:
Indeed, we had this discussion as part of the proposal to reduce the naming scheme to `idr<NNNN>-<name>`. This proposal was rejected or at least postponed  to minimize inconsistency when browsing IDR data with the current entry point.

With the current naming scheme, consortium/project names are used for the description field e.g. `sysgro`, `mitocheck`, `hipsci`, `tara`, `hpa`. I can certainly amend the test and mention this.